src/components/editMeasure/measureDetails/measureInformation/MeasureInformation.tsx

```diff
@@ -297,18 +294,17 @@ export default function MeasureInformation() {
           </div>
         </div>
 
-        <FormButtons>
-          <ButtonWrapper>
-            <SubmitButton
```

Comment:
For little styling like, 
const FormButtons = tw.div`pt-5`;
const ButtonWrapper = tw.div`flex justify-start`;
You can directly add a div and add tw="Styles" as an attribute

Reply:
Rather than, the Element says FormButton, but the actual HTML element is a div